umh-core/Makefile

```diff
@@ -225,7 +225,7 @@ integration-test: stop-all-pods vet nilaway lint
 pre-test: stop-all-pods vet nilaway lint
 
 # Meta target to setup the environment for the tests in CI environments
-pre-test-ci: vet nilaway lint
```

Comment:
This was removed from the integration tests, due to memory constraints.
It is still run on the go unit test, which uses a different runner

Reply:
This is actually part of https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/2070, but this had the wrong base branch.